### DISCUSSION TITLE
1 plot 108.8 gb too big

### DISCUSSION THREAD

**Discussion by elio1454** [Chia-Network/chia-blockchain#8131] [Ideas]
Hi, i see that 1 plot it's 108.8 gb, can you reduce at the maximum the number of gb for a plot, it's too big 108.8 gb .

**Comment by BrandtH22** [comment]
@elio1454 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - the current k-size requirement (k-32) is set to reduce the potential of bad actors plot grinding. It is highly unlikely that this requirement will reduce and more likely that one day the k-size requirement will increase as technology advancements make plot grinding more affordable. To counter this effect, there is now the ability to farm compressed plots which requires more resources to farm a block (the plot files need to be completed at time of farming rather than plotting) but does have the effect of reducing space requirements.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
